app/main.py

```
@@ -349,73 +343,105 @@ def update_jdk_metrics():
+    status = last_build['status']
+    status_value = {"SUCCESS": 1, "FAILURE": 0, "NO_BUILDS": -1}.get(status, -1)
+
+    finish_date = convert_time(last_build.get('finishDate', '')) if status != "NO_BUILDS" else 0
+    start_date = convert_time(last_build.get('startDate', '')) if status != "NO_BUILDS" else 0
+
+    labels = dict(
+        template_id=template_id,
+        build_type_name=cfg["name"],
+        build_type_id=cfg["id"],
+        build_url=cfg["webUrl"],
+    )
```

Ruff (0.15.4)

[convention] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)